node/Cargo.toml

```diff
@@ -0,0 +1,182 @@
+[badges]
+travis-ci = { repository = "centrifuge/centrifuge-chain", branch = "master" }
+maintenance = { status = "actively-developed" }
+is-it-maintained-issue-resolution = { repository = "centrifuge/centrifuge-chain" }
+is-it-maintained-open-issues = { repository = "centrifuge/centrifuge-chain" }
+
+[package]
+name = "centrifuge-chain"
```

Comment:
I wonder whether the crate should rather be named `centrifuge-node`.

Reply:
I see. This was related to my comment about the root toml. Given that the root toml seems to be finde, lets keep `centrifuge-chain` here to not introduce another breaking change.